README.rst

```diff
@@ -24,6 +24,20 @@ Motr
 At the core of `CORTX <https://github.com/Seagate/cortx>`_ lies Motr.  Motr is a distributed object storage system, targeting `mass capacity storage <https://www.seagate.com/products/storage/object-storage-software/>`_
 configurations. To ensure the most efficient storage utilization, Motr interacts directly with block devices (i.e. it does not _layer_ on a local file system).  The Motr design was heavily influenced by the Lustre file system, NFSv4 and database technology. It must be noted that traditional file system properties (hierarchical directory namespace, strong POSIX consistency guarantees, &c.) are no longer desirable or achievable at mass capacity. Instead, Motr is a more general storage system that provides an optional file system interface. This allows wider range of deployments, including cloud.
 
+Following are the features of the Motr submodule:
+
+- Scalable:
```

Comment:
Only this "Scalable" is shown as _ITALIC_.
Why?
Make all them _ITALIC_ and **BOLD**, or make them the same.

Reply:
Sure. Changing